addons/mail/static/tests/document_thread_window_tests.js

```diff
@@ -48,7 +47,7 @@ QUnit.module('Document Thread', {
                 },
                 records: [{
                     id: 1,
-                    author_id: [1, 'Me'],
+                    author_id: [44, 'Me'],
```

Comment:
to minimize the diff, I would just modify to `partnerID` and keep `partnerID` everywhere.